src/main/java/com/andersen/Identifiable.java

```diff
@@ -0,0 +1,12 @@
+package com.andersen;
+
+public interface Identifiable {
```

Comment:
in this case we can use abstract class as well (add getter and setter here, and the field as well)